Documents/Command/AWS-RunAnsible/AWS-RunAnsible.yaml

```diff
@@ -0,0 +1,189 @@
+schemaVersion: '2.2'
+description: 'Use this document to run Ansible playbooks on Amazon EC2 managed instances.
+  Specify either playbook YAML text, playbook URL, or URL. If you specify multiple
+  values, precedence order is: url, PlaybookURL, and then playbook. If you specify
+  URL, everything from that point and below in S3 will be synced to the local machine
+  (including any roles and custom modules). Use PlaybookFile for the playbook to run
+  from that URL. Use the extravar parameter to send runtime variables to the Ansible
+  execution. Use the check parameter to perform a dry run of the Ansible execution.
+  The output of the dry run shows the changes that will be made when the playbook
+  is executed. This version also supports zip files via http or s3 URLs. It also supports
+  providing an s3 URL for a directory containing the playbook files. '
+parameters:
+  playbook:
+    type: String
+    description: (Optional) If you don't specify a URL, then you must specify playbook
+      YAML in this field.
+    default: ''
+    displayType: textarea
+  PlaybookURL:
+    type: String
+    description: '(Optional) If you don''t specify playbook YAML, then you must specify
+      a URL where the playbook is stored. This url can refer to a playbook yaml file,
+      an s3 prefix, or an archive (TAR GZIP/BZIP2, Zip) containing all your playbook
+      artifacts. You can specify the URL in the following formats: http://example.com/playbook.yml
+      or s3://examplebucket/playbook.url. You can also specify an s3 URL for a directory
+      containing the playbooks like s3://mybucket/myplabooks/.  For security reasons,
+      you can''t specify a URL with quotes.'
+    default: ''
+    allowedPattern: ^\s*$|^(http|https|s3)://[^']*$
+  PlaybookFile:
+    type: String
+    description: (Optional) The playbook file to run (including relative path), when
+      specifying a playbook bundle like a zip file or an s3 path to a folder in a
+      bucket
+    default: ssm-playbook.yml
```

Comment:
Can we add input validation, i.e. an allowedPatterns that ensures a .yml file is provided?